src/index.js

```diff
@@ -2,48 +2,82 @@ const es = require("event-stream");
 const fs = require("fs");
 const path = require("path");
 const https = require("https");
+const { execSync, exec } = require('child_process');
+const { EOL } = require('os');
 const { certToPem, pemToCert, defaultFilter } = require("./utils");
 
 const { paths } = require("../res/linux-cert-stores.json");
 const splitPattern = /(?=-----BEGIN\sCERTIFICATE-----)/g;
 const noCertsError = new Error(
   "No certificates were found. If you think this is a bug, please report it at https://github.com/kanishk98/linux-ca/issues"
 );
+const pkcsToken = 'pkcs11:model=p11-kit-trust;manufacturer=PKCS%2311%20Kit;serial=1;token=System%20Trust';
+const p11Args = '--only-urls --batch --list-all-trusted';
 
 const getAllCerts = (readSync = false) => {
   return new Promise((resolve, reject) => {
     const certs = [];
     if (readSync) {
-      for (let certPath of paths)
-        try {
-          certPath = path.resolve(__dirname, certPath);
-          const file = fs.readFileSync(certPath, "utf-8");
-          certs.push(file.split(splitPattern).map(cert => cert));
-          resolve(certs);
-        } catch (err) {
-          if (err.code !== "ENOENT") {
-            reject(err);
+      try {
+        const urls = execSync(`p11tool ${ p11Args } ${ pkcsToken }`, {encoding: 'utf8'});
+        urls.split(EOL).forEach(line => {
+          if (!line){
+            return;
```

Comment:
Work for me: I'll add a git hook to prettify code automatically so no one has to worry about formatting. Didn't expect this tool to be used by anyone, tbh, good problems to have! :)